apps/site/src/_pages/docs/0_index.mdx

```diff
@@ -2,33 +2,41 @@
 
 From a developer perspective, blocks encapsulate a set of specific functionality. Building a block is no different to writing a React, Vue, or [Web Component](https://en.wikipedia.org/wiki/Web_Components) – in fact, these are all ways you might implement a block. They are reusable elements that expect certain data to be passed to them (what we refer to as a block’s ‘schema’). For example, an image block might only accept an image URL in its `src` property, not a number.
 
-## What is a Block Protocol block?
+## What is a Þ block?
 
-Any block can become a Block Protocol block by meeting the requirements defined in the specification. These requirements define what data a block can accept and how they're allowed to edit that data.
+Any block can become a Þ block by meeting the requirements defined in the specification. These requirements define what data a block can accept and how they're allowed to interact with that data. This standardized approach makes it possible to use and embed Þ blocks across any different kind of web application, provided it supports the protocol.
 
-Following these requirements allows BP blocks to do more than regular web blocks. This standardized way of handling data makes it possible to embed BP blocks across different web applications. BP blocks can read and write data to the applications they're embedded within without any extra setup or custom integrations (as long as both the block and embedding application adhere to the specification).
+Þ blocks can read and write data to the applications they're embedded inside without any extra setup or custom integrations (as long as both the block and embedding application adhere to the specification).
 
-BP blocks can also be published on the [Hub](https://blockprotocol.org/hub) – an open-source registry where anyone can publish their blocks for other people to discover and use.
+The Block Protocol is free to use, and any application can enable the use of Þ blocks by following the [requirements for embedders](https://blockprotocol.org/docs/embedding-blocks).
 
-Any application can accept BP blocks, as long as they follow the [requirements for embedding applications](https://blockprotocol.org/docs/embedding-blocks).
+## What is the Þ Hub?
+
+Blocks and types can be published and discovered on the [Þ Hub](https://blockprotocol.org/hub), an open-source registry that anybody can use. There are two ways to access it:
+
+- **in-browser** by navigating to [`blockprotocol.org/hub`](https://blockprotocol.org/hub): no account is required to browse and test blocks
+- **programmatically** from within other applications: either a user or an embedding application must create a free API key in order to query and load blocks from the Þ Hub via its API
```

Comment:
```suggestion
- **programmatically** from within other applications: either a user or an embedding application must create a free API key in order to query blocks from the Þ Hub via its API
```
To clarify that blocks don't require an API key to load – only the search feature does.